### ISSUE TITLE
Translation strings.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#293]
```
Translation strings.xml (r209) to russian


```

Original issue reported on code.google.com by `shell.an...@gmail.com` on 1 Jan 2012 at 2:29

Attachments:
- [strings.xml](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-293/comment-0/strings.xml)

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Few misspelling errors was fixed.
```

Original comment by `shell.an...@gmail.com` on 1 Jan 2012 at 3:34

Attachments:
- [strings.xml](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-293/comment-2/strings.xml)

**Comment by GoogleCodeExporter** [comment]
```
Thanks, looks good. Committed in r213.
```

Original comment by `oskar.wi...@gmail.com` on 6 Jan 2012 at 8:47
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.11**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
